app.py

```diff
@@ -244,6 +244,9 @@ def parse():
     level = int(body ['level'])
     sublevel = body.get ('sublevel') or 0
 
+    code = re.sub(r'\n{2,}', '\n\n', code)
```

Comment:
Why retain even 2 sequential newlines? Wouldn't `\n{1,} -> \n` be even fewer newlines? (And therefore moar better?)